bindings/wysiwyg-wasm/src/lib.rs

```diff
@@ -532,6 +534,7 @@ impl DomHandle {
             wysiwyg::DomNode::Container(node) => node.name().to_string(),
             wysiwyg::DomNode::LineBreak(node) => node.name().to_string(),
             wysiwyg::DomNode::Text(_) => String::from("-text-"),
+            wysiwyg::DomNode::Zwsp(_) => String::from("-text-"),
```

Comment:
Copy & paste error (`-text-`)?

Reply:
Actually I wasn't sure if this string changes something or not for the web implementation. If it doesn't, we probably should set something like `"-zwsp-"` yeah.